scripts/ros_release_python

```diff
@@ -80,7 +81,22 @@ def release_to_debian(name, version, debian_version, upload, ignore_upload_error
             '# ' + ' '.join(cmd) +
             (' [with %s]' % ' '.join(['%s=%s' % (k, v) for k, v in add_env.items()])
              if add_env else ''))
-        subprocess.check_call(cmd, env=dict(os.environ, **add_env))
+        # when doing a Python 2 release but no `python` executable is present
+        # use a temporary symlink to `python2` otherwise `stdeb` fails
+        symlink_python2 = python_version == 2 and not shutil.which('python') and shutil.which('python2')
+        if symlink_python2:
+            tmpdir = tempfile.mkdtemp()
+            # if the Python executable is not in a directory named `bin`
+            # the absolute path is used in the generated shebang lines
```

Comment:
Does the logic generating #! lines come from stdeb, debhelper, or setuptools? I'm asking to gauge which part of the stack could make an internal change which could break this gain.

Reply:
Afaik from debhelper.

Reply:
As long as the distribution's setuptools version is the one used I think that means this behavior could only change when we change distributions. So I'm not too worried about finding a way to be more defensive here.